packages/cli/src/processed-data-managers/index.ts

```diff
@@ -0,0 +1,7 @@
+import { type IProcessedDataManager } from 'n8n-workflow';
+
+import { ProcessedDataHelper } from './processed-data-helper';
+
+export async function getProcessedDataManager(): Promise<IProcessedDataManager> {
+	return new ProcessedDataHelper();
+}
```

Comment:
I don't think this needs to be `async`?

```suggestion
export function getProcessedDataManager(): IProcessedDataManager {
	return new ProcessedDataHelper();
}
```